source/sgp_mode/Tasks.h

```diff
@@ -219,8 +259,35 @@ class TaskSet {
       n_succeeds_sym[i]->store(0);
     }
   }
+
+  /**
+   * Input: A boolean identifying a given organism as a host or a symbiont
+   *
+   * Output: The map of square frequencies that matches the organism's type
+   *
+   * Purpose: Returns either hostCalculationTable or symCalculationTable, depending
+   on whether the organism is a host or a symbiont. 
+   */
+std::map<uint32_t, uint32_t> GetSquareFrequencyData(bool Host){
+  std::map<uint32_t, uint32_t> myMap;
+  if (Host){
+     myMap = tasks[0].hostCalculationTable;
+  }
+  else{
+     myMap = tasks[0].symCalculationTable;
+  }
+  return myMap;
+}
+
+void ClearSquareFrequencyData(){
+  tasks[0].hostCalculationTable.clear();
+  tasks[0].symCalculationTable.clear();
+}
```

Comment:
Looks like your tabs got off, so please go indent the above so that they are clearly within the correct block